OpenSourcePolitics/decidim-module-extended_socio_demographic_authorization_handler
# [Feature] - API proxy

#### Description

Refactor cities autocompletion to reduce API requests.

When not registered, API rate limit is set to 500. This PR allows to define authorization key to have a rate limit equals to 20000/day.

Also, API calls are now performed from backend and response are store in Cache to perform once a request for a given zipcode

#### How to test

* Generate a development_app `bundle exec rake development_app`
* Login as user or admin
* Go to My profile > authorizations
* Select Additional informations
* Type a zipcode and see list refreshed

#### Env vars

`OPENDATASOFT_API_KEY`: If define perform API calls with key
